mods/default/mapgen.lua

```diff
@@ -35,6 +35,7 @@ minetest.register_alias("mapgen_cobble", "default:cobble")
 minetest.register_alias("mapgen_stair_cobble", "stairs:stair_cobble")
 minetest.register_alias("mapgen_mossycobble", "default:mossycobble")
 minetest.register_alias("mapgen_stair_desert_stone", "stairs:stair_desert_stone")
+minetest.register_alias("mapgen_sandstonebrick", "default:sandstonebrick")
```

Comment:
?

Reply:
My goodness, absolutely no idea how that got in there? 0_o Very odd